Bottom: 8378f6714122fbe0e007ad039bfc6b9b197a6a6d
Top:    46072475f6bf59675b34f271105e4a2ba947cf1b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-14 16:24:52 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5..b14d49c4 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
